solidity-v1/dashboard/src/constants/tooltips.js

```diff
@@ -64,6 +64,11 @@ const resourceTooltipProps = {
     redirectLink: "/coverage-pools/how-it-works",
     linkText: "How it works",
   },
+  thresholdPageGrantAllocation: {
+    title: "Grant Allocation",
+    content: "A grant is something that vests KEEP over a set period of time",
```

Comment:
I believe we should use `unlock` term instead of `vest`. Please see https://github.com/keep-network/keep-core/issues/1390 cc @liz-shinn @sashatanase